src/index.js

```diff
@@ -45,8 +45,7 @@ class FontAwesome extends React.Component {
       stack,
       tag = 'span',
       ariaLabel,
-      ...props,
-    } = this.props
+      ...props } = this.props
```

Comment:
About this change, it's a little problematic, since it brakes the linting, apparently it should not have a trailing comma but your linting configuration enforces it, that's a work-around in order to keep the linter happy...